### PR TITLE
Cleanup and added a TC in the file.

### PR DESCRIPTION
**Pull request by amitsrivastava78**
Did some cleanup and added the TC for the transformation.

### CODE REVIEW

File: tensorflow/lite/toco/graph_transformations/tests/identify_lstm_test.cc [hunk 1]
```diff
@@ -0,0 +1,172 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```
Comment (jianlijianli) [inline tensorflow/lite/toco/graph_transformations/tests/identify_lstm_test.cc]: This should be 2019 :)

**Review by jianlijianli — DISMISSED** [review]
Thanks for making the change. Just a minor comment.

**Comment by gbaned** [comment]
@amitsrivastava78 can you please check ubuntu sanity errors ? Thanks!

File: tensorflow/lite/toco/graph_transformations/tests/identify_lstm_test.cc (hunk 1 above)
Comment (amitsrivastava78) [reply tensorflow/lite/toco/graph_transformations/tests/identify_lstm_test.cc]: @jianlijianli , thanks for the review i have updated your comments along with removing of the logging dependency from the BUILD file, kindly check and approve.

Sorry for the inconvenience caused.

Regards
Amit

**Comment by amitsrivastava78** [comment]
> @amitsrivastava78 can you please check ubuntu sanity errors ? Thanks!
@gbaned , thanks for pointing this out, i have fixed the problem.

Regards
Amit

**Review by jianlijianli — DISMISSED** [review]
Thanks.

**Comment by amitsrivastava78** [comment]
@gbaned , all the checks have passed for this PR, can you please help to merge this.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@gbaned , all the checks have passed for this PR, can you please help to merge this.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@gbaned , can you please help to merge this PR.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@gbaned , can you please help to get this merged.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@jianlijianli and @miaout17 I have resolved the merge conflicts , can you please have alook.


Regards
Amit

**Comment by gbaned** [comment]
@amitsrivastava78 Could you please resolve the conflicts? Thanks!

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**